federatedscope/nlp/configs/cfg_client.py

```diff
@@ -0,0 +1,19 @@
+import os.path as osp
+import copy
+
+
+def extend_cfg_client(init_cfg, cfg_client):
+    num_clients = len([k for k in cfg_client.keys() if k.startswith('client')])
+    for i in range(1, num_clients + 1):
+        cfg = cfg_client['client_{}'.format(i)]
+        task = cfg.data.type
+        cfg.data.batch_size = init_cfg.data.all_batch_size[task]
```

Comment:
It seems like your want to set the `batch_size` according to the type of tasks. 
For now, the type of task in each client is decided by the dataloader, which means we know already know the task type of each client. Therefore, we can set the cfg_client as follows to set different parameters. 
```
client_1:
    data:
        batch_size: 20
client_2:
    data:
        batch_size: 30
client_3:
    data:
        batch_size: 50
```